mysys/CMakeLists.txt

```diff
@@ -138,9 +144,23 @@ ELSEIF(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64")
       IF(HAVE_ARMV8_CRYPTO)
         ADD_DEFINITIONS(-DHAVE_ARMV8_CRYPTO)
       ENDIF()
+      IF(NOT HAVE_ARMV8_NEON)
+        set(CMAKE_REQUIRED_FLAGS "-mfpu=neon -mfloat-abi=softfp")
+
+        CHECK_CXX_SOURCE_COMPILES("
+        #include <arm_neon.h>
+        int main(){}" HAVE_ARMV8_NEON_SOFTFP)
```

Comment:
`SET(CMAKE_REQUIRED_FLAGS)` required here now that the test is finished.

Reply:
DO you mean it should be turned off?

Reply:
Yes. I did. As you made it. Good.